admin-tools/data_mover.sh

```diff
@@ -0,0 +1,116 @@
+#! /bin/bash
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of fiware-cosmos.
+#
+# fiware-cosmos is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# fiware-cosmos is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with fiware-cosmos. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# francisco dot romerobueno at telefonica dot com
+
+# Show the usage
+if [ $# -ne 4 ]; then
+	echo "Usage: ./data-mover.sh <usernames_file> <src_cluster_URL> <dst_cluster_URL> <max_bytes_transfer>"
+	exit 1;
+fi
+
+# Arguments
+usernamesFile=$1
+srcClusterURL=$2
+dstClusterURL=$3
+maxBytesTransfer=$4
+
+# Create a temporal working directory
+tmpDir=$(mktemp -d /tmp/data_mover.XXXX)
+
+# Migrates a given path
+# $1 : path to be migrated
+# $2 : username owning of the path
+migrate() {
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/4b5d2ac73b07fde405d6014ee14c34fbb2071b77 and https://github.com/telefonicaid/fiware-cosmos/commit/ca3edcf8a7307281bef7bb6e6c9176cc763b7203